Backend/src/main/java/idealab/api/model/Auditable.java

```diff
@@ -0,0 +1,52 @@
+package idealab.api.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@MappedSuperclass
+public class Auditable {
```

Comment:
Can you change this class to `RecordTimestamp`.  We are going to use audits somewhere else and don't want to get confused.

Reply:
Sure @jberns I will change tonight